### PR TITLE
example to show the difference between "sum by" and "sum without"

### PR DESCRIPTION
**Pull request by krasi-georgiev**
the official description is not bad , but I understood the difference only when I looked at an example from an old Julius's post

### CODE REVIEW

File: docs/querying/operators.md [hunk 1]
```diff
@@ -218,6 +218,10 @@ number of seen HTTP requests per application and group over all instances via:
 
     sum(http_requests_total) without (instance)
 
+Which is equivalent to:
+
+    sum(http_requests_total) by (application,group)
```
Comment (brian-brazil) [inline docs/querying/operators.md]: Space after comma

Comment (krasi-georgiev) [reply docs/querying/operators.md]: updated.

Feel free to change anything else that doesn't look right.

**Comment by brian-brazil** [comment]
Hmm, that didn't work. Can you rebase this to on top of release-2.0?

**Comment by krasi-georgiev** [comment]
@brian-brazil  is it ok now?

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**